docs/blog/posts/community/2024/2024-01-04-meeting.md

```diff
@@ -0,0 +1,92 @@
+---
+date: 2024-01-04
+authors:
+  - billy99
+categories:
+  - Community Meeting
+  - "2024"
+---
+
+# Community Meeting: January 4, 2024
+
+## Welcome to 2024!
+
+Welcome to the first `bpfman` Community Meeting of 2024.
+We are happy to start off a new year and excited for all the changes in store
+for `bpfman` in 2024!
+
+Below were some of the discussion points from this weeks Community Meeting.
+
+* bpfman-csi Needs To Become Its Own Binary
+* Kubernetes Support For Attaching uprobes In Containers
+* Building The Community
+
+<!-- more -->
+
+## bpfman-csi Needs To Become Its Own Binary
+
+Some of the next work items for `bpfman` revolve around removing the async code
+from the code base, make `bpfman-core` a rust library, and removing all the
+gRPC logic.
+Dave is currently investigating this.
+One area to help out is to take the `bpfman-csi` thread and making it it's own
+binary.
+This may require making `bpfman` a bin and lib crate (which is fine, just needs
+a lib.rs and to be very careful about what we’re exporting).
+Andrew is starting to take a look at this.
```

Comment:
Github tags would be better here but this is fine for now 👍

Reply:
Done